src/frontend/screens/Game/GamePage/index.css

```diff
@@ -396,6 +395,7 @@
     .anticheatInfo {
       max-width: none;
       margin: 0;
+      align-self: center;
```

Comment:
anticheat info in game page looked misaligned and too close to the div above

![image](https://github.com/user-attachments/assets/d7b0ad46-b3e9-4698-9a0f-5d9fcc951cff)

and now:
![image](https://github.com/user-attachments/assets/119299ef-afef-409e-b33f-e72d99e80031)